server.go

```diff
@@ -71,6 +72,35 @@ type bug struct {
 	PointValue int    `json:"pointValue"`
 }
 
+type scoring_alert struct {
+	RecentHits []string `json:"recent_hits"` // encoded scoring message
+}
+
+type scoring_message struct {
+	RepoOwner   string         `json:"repositoryOwner"`
+	RepoName    string         `json:"repositoryName"`
+	TriggerUser string         `json:"triggerUser"`
+	TotalFixed  int            `json:"fixed-bugs"`
+	BugCounts   map[string]int `json:"fixed-bug-types"`
+	PullRequest int            `json:"pullRequestId"`
+}
+
+type leaderboard_item struct {
+	UserName string `json:"name"`
+	Slug     string `json:"slug"`
+	Score    int    `json:"score"`
+	Archived bool   `json:"_archived"`
+	Draft    bool   `json:"_draft"`
+}
+
+type leaderboard_payload struct {
+	Fields leaderboard_item `json:"fields"`
+}
+
+type leaderboard_response struct {
```

Comment:
I think snake case is kinda "not a golang thing". I'd do `leaderboardResponse` or `LeaderboardResponse` if you need it to be public (it would be private to main in this current convention)

Reply:
done 22dcf90